docs/guide.rst

```diff
@@ -2,4 +2,76 @@
 User Guide
 **********
 
-This user guide provides a walkthrough of the major features in the ``hvpy`` package.
+This guide provides a walkthrough of the major features in the ``hvpy`` package.
+``hvpy`` is a python wrapper for the `Helioviewer API <https://api.helioviewer.org/docs/v2/>`__, which is the same API that powers `helioviewer.org <https://helioviewer.org>`__.
+
+Basic Usage
+-----------
+Each API endpoint has a function in ``hvpy`` that shares the same name and has the same input parameters.
+For example if you are looking at `getJP2Image <https://api.helioviewer.org/docs/v2/api/api_groups/jpeg2000.html#getjp2image>`__ in the Helioviewer documentation, you would execute this in ``hvpy`` with the following command:
+
+.. code-block:: Python
+
+    import hvpy
+    from hvpy.datasource import DataSource
+    from datetime import datetime
+
+    jp2 = hvpy.getJP2Image(datetime.now(), DataSource.AIA_94.value)
```

Comment:
```suggestion
    >>> from datetime import datetime
    >>> import hvpy
    >>> from hvpy.datasource import DataSource

    >>> jp2 = hvpy.getJP2Image(datetime.now(), DataSource.AIA_94.value)
```

I wonder if this will run the code as an experiment